### PR TITLE
Rename database fields from snake_case to camelCase

### PR DESCRIPTION
**Pull request by danielrozenberg**
Quick refactor

### CODE REVIEW

File: test-status/web.js [hunk 1]
```diff
@@ -142,19 +142,19 @@ exports.installWebUiRouter = (app, db) => {
     const errors = validationResult(request);
     if (!errors.isEmpty()) {
       response.render('status', Object.assign({
-        short_head_sha: request.short_head_sha,
+        shortHeadSha: request.shortHeadSha,
         is_skipping: true,
```
Comment (rsimha) [inline test-status/web.js]: Is `is_skipping` something that needs to be changed too?

File: test-status/api.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ function createNewCheckParams(pullRequestSnapshot, type, subType, status) {
     owner: pullRequestSnapshot.owner,
     repo: pullRequestSnapshot.repo,
     name: `ampproject/tests/${type} (${subType})`,
-    head_sha: pullRequestSnapshot.head_sha,
+    head_sha: pullRequestSnapshot.headSha,
```
Comment (rsimha) [inline test-status/api.js]: How come `head_sha` remains in snake case?

File: test-status/web.js (hunk 1 above)
Comment (danielrozenberg) [reply test-status/web.js]: This isn't a db variable, it's a template variable, but might as well! Done.

File: test-status/api.js (hunk 2 above)
Comment (danielrozenberg) [reply test-status/api.js]: This is for the GitHub API, which is not using the same style guide as ours. I thought about making some extra compatibility layer for a hot second and decided it's not worth the hassle 😅

**Pull request merged**